docs/paas-plus/dev/platform-as-a-service-plus.md

```diff
@@ -0,0 +1,161 @@
+
+<img width="261" alt="image1" src="https://user-images.githubusercontent.com/83701393/176354590-de6b3af1-4f38-414f-9391-2731cb078402.png">
+
+
+### Spryker CI overview
+
+Spryker CI is powered by [Buddy.works](https://buddy.works), and provides an intuitive UI for a seamless CI/CD experience.
+
+It comes with two major functionalities: 
+- Automated upgrades
+- Quality checks
+
+Spryker's new upgrader service provides effortless upgrades for minor and patch releases. The new CI pipeline will also provide automated checks, a quality gate, that guides Spryker developers to ensure compliance with Spyker best practices and further fuel upgradability success.
+
+## Onboarding to PaaS Upgradability
+Onboarding to PaaS+ happens in two steps: 
+1. getting your project ready and compliant with a set of pre-requirements, to ensure you fully benefit from all the features in our product
+2. getting started in Spryker CI
+
+
+## Step 1: Getting your project ready for PaaS Upgradability  
+For our current version of the upgradability package, the following pre-requisites apply:
+
+//need better formating for the pre-requisites
+
+**1. Customer should be on the latest major version available for a module** 
+**Why?** Spryker upgrader service will automatically provide minor and patches. If your application is not on the latest major of a module, you will only be able to receive minor and patches that apply to the latest major version you have upgraded to.
+
+**2. Customer should be compliant with Spryker Quality Gate**
+**-- Why?** Spryker Quality Gate contains code checkers to ensure code compliance with Srpyker development and customization best practices. These code checkers ensure that your Spryker application is compatible with minor and patch updated, even when it’s highly customized. 
+**How can I know if my project is compliant?** Please read our documentation on [Keeping a project upgradable] (https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/keeping-a-project-upgradable.html#select-a-development-strategy). You can check if your customized project is compliant with Spryker Quality Gate by using our [tool] (https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/keeping-a-project-upgradable.html#check-if-project-is-upgradable-using-the-evaluator-tool). Follow our [guidelines] (https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.html) in order to solve any non-compliance issues.
+
+**3. Customer should develop their Spryker application using Spryker SDK**
+**Why?** Customers need to use Spryker SDK throughout the development life cycle, to ensure compliance with Spryker architecture and customization best practices. 
```

Comment:
This is a prerequisite. At which point should they start using Spryker SDK?

Reply:
Ideally, they should start using SDK from the start of the project. But, if they want to onboard to PaaS+, then at that point they need to use SDK throughout their development. 

Does it make sense? Let me know if I should provide further explanation